openpass-js-sdk/src/app.js

```diff
@@ -26,14 +26,18 @@ async function startServer() {
         return res.render("redirect-manual-button.html");
     });
 
-    app.get("/quick-auth-html", (_, res) => {
-        return res.render("quick-auth-html.html");
+    app.get("/quick-auth-html-popup", (_, res) => {
```

Comment:
Maybe rename the non html one to js-api like the sign in button?

Reply:
Done